packages/angular/migrations.json

```diff
@@ -362,6 +293,15 @@
       "version": "21.0.0-beta.3",
       "description": "Set the `continuous` option to `true` for continuous tasks.",
       "factory": "./src/migrations/update-21-0-0/set-continuous-option"
+    },
+    "change-data-persistence-operators-imports-to-ngrx-router-store-data-persistence": {
```

Comment:
This one duplicates a migration that runs for Nx 16.2, but it's needed if someone has opted out of it in earlier versions. Since we're removing the operators in Nx v21, we must ensure we migrate those cases.